test/common/test_api_utils.py

```diff
@@ -404,7 +404,7 @@ def get_supported_pollers(node_id):
 
     if mondata['status'] not in [200,201,202,204]:
         if fit_common.VERBOSITY >= 2:
-            print "Status: {},  Failed to get data for poller id: {}".format(mondata['status'], str(poller_id))
+            print "Status: {},  Failed to get poller data for node: {}".format(mondata['status'], node_id)
```

Comment:
Sounds picky, but since you are changing it, I think this code be confusing.  I think it should say "Failed to get pollers for node".
The API above is getting the pollers for a node.  Then there is a different API for getting the poller data for a particular poller.